db/migrate/20200205225813_add_notifiable_id_notifiable_type_action_index_to_notifications.rb

```diff
@@ -0,0 +1,10 @@
+class AddNotifiableIdNotifiableTypeActionIndexToNotifications < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    add_index :notifications, %i[notifiable_id notifiable_type action],
+              unique: false,
+              name: "index_notifications_on_notifiable_id_notifiable_type_and_action",
+              algorithm: :concurrently
```

Comment:
👏 for remembering to add `:concurrently`, came here to suggest that only to find it's already there 👍